contracts/eth/nearprover/utils/upgrade_admin.js

```diff
@@ -20,10 +20,11 @@ async function upgradeAdminAddressTo ({
     adminAddressSlot,
     ledgerKeyPath,
 }) {
-    const currentAdminAtSlot = ethers.BigNumber.from(await provider.getStorageAt(contractAddress, Number(adminAddressSlot))).toHexString();
+    const slotContent = await provider.getStorageAt(contractAddress, Number(adminAddressSlot));
+    const currentAdminAtSlot = ethers.BigNumber.from(slotContent).toHexString();
```

Comment:
it's just slot content, we shouldn't expect by default that there will be an admin, in this case, this is exactly the purpose of all those checks. We rather should signalize that admin is not in the selected slot and the current slot content is the following (printing the content)

Reply:
fixed at 86fbac9f5b3286500a4206fd795e1cb695494740, please check